.openshift-ci/drivers/scripts/push-drivers.sh

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env bash
 
-set -eo pipefail
+set -exo pipefail
```

Comment:
Is `-x` needed, maybe a leftover?

Reply:
It's just foe debug, I'll remove it before merging